default.properties

```diff
@@ -1,8 +1,24 @@
 # This file holds some property shared across all geOrchestra webapps
 
+# Scheme of the geOrchestra instance
+# Should be https
+scheme=https
+
+# Domain name of the geOrchestra instance
+# Once modified, adapt the following files accordingly:
+# - mapfishapp/wfs.servers.json
+# - mapfishapp/wms.servers.json
+# - mapfishapp/wmts.servers.json
+# - mapfishapp/credentials.properties
+# - mapfishapp/print/config.yaml
+# - mapfishapp/js/GEOR_custom.js
+# - ...
+# or replace all the strings with `sed` (see README.md)
+domainName=georchestra.mydomain.org
+
 # Public URL of this geOrchestra instance
 # URL must not include the trailing slash
-publicUrl=https://georchestra.mydomain.org
+publicUrl=${scheme}://${domain}
```

Comment:
As a result, there's no use having `publicUrl` in default.properties ?

Reply:
Caution also with domain / domainName

Reply:
> As a result, there's no use having `publicUrl` in default.properties ?

The use is to avoid repeating `${scheme}://${domain}` in the properties files... There are about 20 occurrences of publicUrl in these files, it's better to factorize.